examples/benchmark.py

```diff
@@ -10,7 +10,7 @@
 import numpy as np
 
 parser = argparse.ArgumentParser("Running benchmarks on simulator")
-parser.add_argument("--scene", type=str, default="test.glb")
+parser.add_argument("--scene", type=str, required=True)
```

Comment:
Let's keep a default for this arguments and make it one of the habitat-test-scenes, say `data/scene_datasets/habitat-test-scenes/skokloster-castle.glb`

Reply:
Additionally, if none is specified, can we auto download and extract for them?